exercises/beer-song/canonical-data.json

```diff
@@ -1,49 +1,77 @@
-      }
-    ]
-  }
+  "exercise": "beer-run",
```

Comment:
meh, not sure why I put that in there, I know it is beer-song.  I can fix it now